spec/models/order_spec.rb

```diff
@@ -9,6 +9,10 @@
     it { should validate_presence_of :zip }
   end
 
+  describe "enum" do
+    it { should define_enum_for(:status).with_values([:pending, :packaged, :shipped, :cancelled]) }
+  end
```

Comment:
Testing new migration! Did not know about this-- very cool stuff! enum is pretty magical